src/lib/declarations.ts

```diff
@@ -290,6 +290,16 @@ export const EXCLUDED_KEY_BINDS = [
   "AltRight",
 ];
 
+/**
+ * The set of modifers that will be checked to determine if a given key should
+ * be ignored when handling key events.
+ *
+ * Note that these correspond to `KeyboardEvent.key` values since `KeyboardEvent.getModifierState`
+ * does not accept `KeyboardEvent.code` values.
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState}
```

Comment:
I mainly thought to point this out since  `EXCLUDED_KEY_BINDS` uses `KeyboardEvent.code` values and I wanted to make it clear that these values are a bit different since they use `KeyboardEvent.key` values.